spec/features/user_is_creating_multiple_sponsors_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.feature 'User enters new sponsor data', :type => :feature do
+
+  background do
+    an_agent_exists
+    i_sign_in_as_admin
+  end
+
+  scenario 'User clicks the Create and Add Another button' do
+    when_i_enter_new_sponsor_info
+    and_i_click_create_and_add_another_button
+
+    then_the_sponsor_should_be_saved
+    and_i_should_be_on_new_sponsor_page
+  end
+end
+
+def an_agent_exists
+  FactoryGirl.create :agent
+end
+
+def i_sign_in_as_admin
+  admin = FactoryGirl.create :admin_user
+  visit new_admin_user_session_path
+  fill_in 'Email', with: admin.email
+  fill_in 'Password', with: admin.password
+  click_button 'Login'
+end
+
+def when_i_enter_new_sponsor_info
+  visit new_hq_sponsor_path
+  fill_in 'Name', with: 'C. Montgomery Burns'
+  fill_in 'Requested orphan count', with: '1'
+  select 'London', from: 'sponsor[branch_id]'
+  fill_in 'New city name', with: 'London'
+  select User.first.user_name, from: 'sponsor[agent_id]'
+end
+
+def and_i_click_create_and_add_another_button
+  click_button 'Create and Add Another'
+end
+
+def then_the_sponsor_should_be_saved
+  expect(page).to have_text 'Sponsor successfuly created'
+  expect(Sponsor.last.name).to eq 'C. Montgomery Burns'
+end
+
+def and_i_should_be_on_new_sponsor_page
+  expect(page).to have_text 'New Sponsor'
+end
```

Comment:
Not sure about the file name(s), I suppose whatever we agree on. The nice thing here vs. Cucumber is that only some steps can be defined in common files and thus be accessible from every feature, but those specific to a particular feature can continue to live right below the feature in the same file. 

I also propose that we keep features short and specific, whether we write them in RSpec or Cucumber - like here, it's not a generic "sponsors_spec.rb" that catches all sponsor-related tests, but a focused "user_is_creating_multiple_sponsors_spec.rb".